build.gradle

```diff
@@ -91,4 +92,11 @@ bintray {
         labels = ['base crm', 'api', 'http', 'v2', 'client', 'HTTP client']
         publicDownloadNumbers = true
     }
+}
+
+test {
+    if (!System.getenv('BASECRM_ACCESS_TOKEN')) {
```

Comment:
We shall not by default run tests that require external dependencies (e.g. Base CRM application running and test account on it). By default only unit tests shall be running which should not require Internet connection or any other external dependency.